arkane/ess/qchem.py

```diff
@@ -272,9 +272,10 @@ def load_conformer(self, symmetry=None, spin_multiplicity=0, optical_isomers=Non
 
     def load_energy(self, zpe_scale_factor=1.):
         """
-        Load the energy in J/mol from a QChem log file. Only the last energy
-        in the file is returned. The zero-point energy is *not* included in
-        the returned value.
+        Load the energy in J/mol from a QChem log file. Prioritize the energy from a converged
+        geometry optimization (assigned to variable "a"). If the file does not contain an optimization job
+        or if the optimization hit the maximum cycles, return the next best energy, such as from a frequency job
```

Comment:
Actually, maybe change "return the next best source" to "return the next equivalent source"

Reply:
Updated the docstring :)